lib/ffe.gd

```diff
@@ -30,15 +31,19 @@
 ##  <P/>
 ##  &GAP; can represent elements of all finite fields
 ##  <C>GF(<A>p^d</A>)</C> such that either
-##  (1) <A>p^d</A> <M>&lt;= 65536</M> (in which case an extremely efficient
-##      internal representation is used);
+##  (1) <A>p^d</A> <M>&lt;=</M> <C>MAXSIZE_GF_INTERNAL</C> (in which case an 
+##      extremely efficient internal representation is used);
```

Comment:
Super nitpick (and I know this is old text you just reformatted, so feel free to ignore): Is the word "extremely" really appropriate? Maybe "highly"?